integrations/scikit-learn/template/{{ cookiecutter.module_name }}/README.md

```diff
@@ -0,0 +1,65 @@
+---
+title: {{ cookiecutter.module_name }}
+emoji: 🤗 
+colorFrom: blue
+colorTo: orange
+tags:
+- evaluate
+- {{ cookiecutter.module_type }}
+- sklearn
+description: >-
+  "{{ cookiecutter.docstring_first_line }}"
+sdk: gradio
+sdk_version: 3.12.0
+app_file: app.py
+pinned: false
+---
+
+This metric is part of the Scikit-learn integration into 🤗 Evaluate. You can find all available metrics in the [Scikit-learn organization](https://huggingface.co/scikit-learn) on the Hugging Face Hub.
```

Comment:
Maybe would be nice to also put a redirection to the scikit-learn documentation page of the metric in context since their documentation is very in-depth usually and contains further notes for problem a user is solving, WDYT?

Reply:
e.g. https://scikit-learn.org/stable/modules/generated/sklearn.metrics.explained_variance_score.html#sklearn.metrics.explained_variance_score